composer.json

```diff
@@ -1,15 +1,15 @@
 {
-    "name": "joaopaulolndev/filament-general-settings",
+    "name": "untillnesss/filament-general-settings",
     "description": "Filament package to manage general settings",
     "keywords": [
-        "joaopaulolndev",
+        "untillnesss",
         "laravel",
```

Comment:
Why are you changing to your name the package?

Reply:
Opps, my bad. I just wanted to use this package locally using Composer, but it turns out the changes came here too.